app/uk/gov/hmrc/brm/config/Features.scala

```diff
@@ -102,59 +106,74 @@ trait FeatureFactory {
 object NRSConcreteFeature extends FeatureFactory {
 
   def feature: Boolean = NRSFeature().enabled()
+
   def referenceFeature: Boolean = NRSReferenceFeature().enabled()
+
   def detailsFeature: Boolean = NRSDetailsFeature().enabled()
 
-  override def isDetailsMatchingEnabled(implicit p: Payload) : Boolean = {
+  override def isDetailsMatchingEnabled(implicit p: Payload): Boolean = {
     detailsFeature
   }
 
-  override def isReferenceMatchingEnabled(implicit p: Payload) : Boolean = {
+  override def isReferenceMatchingEnabled(implicit p: Payload): Boolean = {
     referenceFeature
   }
 
   override def referenceFeatures(implicit p: Payload) = validDateOfBirth && isReferenceMatchingEnabled
+
   override def detailsFeatures(implicit p: Payload) = validDateOfBirth && isDetailsMatchingEnabled
 }
 
 object GROConcreteFeature extends FeatureFactory {
 
   def feature: Boolean = GROFeature().enabled()
+
   def referenceFeature: Boolean = GROReferenceFeature().enabled()
+
   def detailsFeature: Boolean = GRODetailsFeature().enabled()
 
-  override def isDetailsMatchingEnabled(implicit p: Payload) : Boolean = {
+  override def isDetailsMatchingEnabled(implicit p: Payload): Boolean = {
     detailsFeature
   }
 
-  override def isReferenceMatchingEnabled(implicit p: Payload) : Boolean = {
+  override def isReferenceMatchingEnabled(implicit p: Payload): Boolean = {
     referenceFeature
   }
 
   override def referenceFeatures(implicit p: Payload) = validDateOfBirth && isReferenceMatchingEnabled
+
   override def detailsFeatures(implicit p: Payload) = validDateOfBirth && isDetailsMatchingEnabled
 }
 
+case class GRONIFeature() extends Feature("groni")
+
+case class GRONIReferenceFeature() extends Feature("groni.reference")
+
+case class GRONIDetailsFeature() extends Feature("groni.details")
+
 object GRONIConcreteFeature extends FeatureFactory {
-  def feature: Boolean = false
-  def referenceFeature: Boolean = false
-  def detailsFeature: Boolean = false
+  def feature: Boolean = GRONIFeature().enabled()
 
-  override def isDetailsMatchingEnabled(implicit p: Payload) : Boolean = {
-    true
+  def referenceFeature: Boolean = GRONIReferenceFeature().enabled()
+
+  def detailsFeature: Boolean = GRONIDetailsFeature().enabled()
+
+  override def isDetailsMatchingEnabled(implicit p: Payload): Boolean = {
+    detailsFeature
   }
 
-  override def isReferenceMatchingEnabled(implicit p: Payload) : Boolean = {
-    true
+  override def isReferenceMatchingEnabled(implicit p: Payload): Boolean = {
+    referenceFeature
   }
```

Comment:
I've removed these from the interface and the other concrete implementations. Called directly now.